packages/mdc-list/index.js

```diff
@@ -160,8 +160,10 @@ class MDCList extends MDCComponent {
 
   initializeListType() {
     // Automatically set single selection if selected/activated classes are present.
-    const preselectedElement =
-      this.root_.querySelector(`.${cssClasses.LIST_ITEM_ACTIVATED_CLASS}, .${cssClasses.LIST_ITEM_SELECTED_CLASS}`);
+    const preSelectedSelector = `.${cssClasses.LIST_ITEM_ACTIVATED_CLASS},
+                                    .${cssClasses.LIST_ITEM_SELECTED_CLASS},
+                                    ${strings.IS_ARIA_CHECKED}`;
```

Comment:
I don't think we want this change... won't this automatically set singleSelection for lists containing radios or checkboxes? We definitely don't want this for checkbox lists, and I'm not sure if we even want it for radio lists right now.

cc @williamernest